procedures/crates.md

```diff
@@ -103,28 +103,40 @@ In summary, the process for establishing an out-of-tree crate is as follows:
 
 1. Where appropriate, discuss and confirm the need within the working group for the out-of-tree
    crate.
-2. Create a PR modifying this document to include the crate in the list below. Use
+1. Create a PR modifying this document to include the crate in the list below. Use
    [`@rfcbot merge`](https://github.com/anp/rfcbot-rs#usage) to gain agreement from compiler
    team members.
-3. Create a new repository in the `rust-lang` organization (permissions should be available for all
-   compiler team members).
-4. Add a README describing the intended purpose of the crate, which team and working group are
-   responsible (link to their page in this repository) and the intended level of maintenance and
-   stability.
-
-   > This crate is developed and maintained by the [Rust compiler team](..) for use within
-   > `rustc`, in particular, it is the responsibility of the
-   > [`.template`](../working-groups/.template) working group. This crate [will have regular
-   > breaking changes and provides no stability guarantees|is intended to remain stable and have
-   > limited breaking changes].
-5. Include the [LICENSE-APACHE][apache] and [LICENSE-MIT][mit] files from `rust-lang/rust`.
-6. Include or link the [CODE_OF_CONDUCT][coc] file from `rust-lang/rust`.
-7. Consult with the infrastructure team to set up `@bors` and `@rust-highfive` and a process for
+1. Create a new repository in the `rust-lang` organization.
+  1. Navigate to `Settings > Collaborators & Teams` and add the `Compiler` team with admin
```

Comment:
I think this should be *write* privileges
```suggestion
  1. Navigate to `Settings > Collaborators & Teams` and add the `Compiler` team with write
```

Reply:
If I'm not mistaken, nobody needs admin privileges (right @pietroalbini?)

Reply:
Nope, the only thing admin privileges are needed for are the settings panel and bypassing branch protection -- requiring an org owner for those actions is fine, we don't do them that often.